citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -949,6 +949,11 @@ protected void doPostJsonResponse(ParameterParser params,
 				this.captureAccess(params, state, edit, results);
 				this.captureAvailability(params, edit, results);
 				getContentService().commitResource(edit, priority);
+
```

Comment:
Why is this the correct place to set the mime type on the pipe rather than somewhere like `doFinish()`?